CleverRaven/Cataclysm-DDA
# Fix bad ter error temple_stairs and temple_finale

#### Summary
None


#### Purpose of change
- Fix #61050
Avoid bad ter error that may occur when loading saved data created before Aug. 2022.


#### Describe the solution
Convert old ters( temple_stairs and temple_finale ) to modern ter


#### Describe alternatives you've considered
Tolerate the occurrence of bad ter error and let the strange temples disappear.


#### Testing
1. Load game
2. Wait until the time when the bad ter error should have happened
3. Confirm no errors occur past the time
4. Go to the strange temple
5. Also confirm the strange temple is existing( if you visit after the error, the remains should have disappeared. )
6. **Note** : If you go near a strange temple without waiting until the time of the bad ter error was supposed to occur, **it will crash**


#### Additional context
- Created with reference to #61570.
For a long time I did not know how to avoid this error, but now found that the PRs' fixes method above could be applied and created the fix.

**I have no knowledge of map generation mechanism, so please point out if this is a misguided fix.**
